src/frontend/src/App.tsx

```diff
@@ -120,7 +120,7 @@ export default function App() {
             <FetchErrorComponent
               description={FETCH_ERROR_DESCRIPION}
               message={FETCH_ERROR_MESSAGE}
-              openModal={isErrorHealth}
+              openModal={isErrorHealth || (healthData && Object.values(healthData).some((value) => value !== "ok"))}
```

Comment:
shouldn't `isErrorHealth` be == `Object.values(healthData).some((value) => value !== "ok")`?

Reply:
that's what the second part of the line 123 does, the (healthData && Object.values(healthData).some((value) => value !== "ok"))

Reply:
Yes but shouldn't the `isErrorHealth` should be this. What is isErrorHealth? if any of the values in healthData is not ok it is an error, correct?

Reply:
isErrorHealth is if the http request fails